src/inputs/input.scss

```diff
@@ -73,3 +72,19 @@
       linear-gradient(t(iui-color-background-1), t(iui-color-background-1));
   }
 }
+
+@mixin iui-input-sizes {
```

Comment:
If you want to have mixin for this, maybe better do it with param and have only padding and min-height rules here.

Reply:
I removed the mixin.